src/cljs/athens/views.cljs

```diff
@@ -16,9 +16,13 @@
     (set! (.-onload fr) #(dispatch [:parse-datoms (.. % -target -result)]))
     (.readAsText fr file)))
 
+(defn- date-string [x] (if (< x 1)
+                         [:span (style/+unknown-date {}) "(unknown date)"]
```

Comment:
I wonder where this is coming from. Is this a problem of the import data?

Reply:
small note, I think `cljsstyle` doesn't catch this, but the `(if ...)` should be on the next line

Reply:
<img width="896" alt="Athens_Research" src="https://user-images.githubusercontent.com/27645/82253026-748c3300-9950-11ea-950e-bcfae32b01a2.png">

The pages that don't have a date are pages of a day (from Daily Notes_